### PR TITLE
Ripe browser destination

### PR DESCRIPTION
**Pull request by simpajj**
This is the Ripe browser destination implementation, as part of the December 2022 Cohort. 

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

**Comment by dlasky** [comment]
Actions tester recently added browser support

**Comment by simpajj** [comment]
> Actions tester recently added browser support

Thanks!  Found it and tested it end-to-end now.

File: packages/browser-destinations/src/destinations/ripe/group/__tests__/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      group: jest.fn().mockResolvedValueOnce(undefined),
+      setIds: jest.fn().mockResolvedValueOnce(undefined)
+    } as unknown as RipeSDK
+
+    const [event] = await RipeDestination({
+      subscriptions,
+      apiKey: '123'
+    })
+
+    const ajs = new Analytics({ writeKey: '123' })
```
Comment (silesky) [inline packages/browser-destinations/src/destinations/ripe/group/__tests__/index.test.ts:43]: Hmmm, the class that takes a writeKey is AnalyticsBrowser, I'm surprised this compiles.

Comment (simpajj) [reply packages/browser-destinations/src/destinations/ripe/group/__tests__/index.test.ts:43]: Hmm, the AnalyticsSettings interface looks like this:

```typescript
export interface AnalyticsSettings {
  writeKey: string
  timeout?: number
  plugins?: Plugin[]
  [key: string]: unknown
}
```

Comment (simpajj) [reply packages/browser-destinations/src/destinations/ripe/group/__tests__/index.test.ts:43]: @silesky Happy to reconsider my testing approach. Looks like most of the other browser destinations do something similar. Any suggestions?

Comment (silesky) [reply packages/browser-destinations/src/destinations/ripe/group/__tests__/index.test.ts:43]: I stand corrected! Carry on 😄

File: packages/browser-destinations/src/destinations/ripe/index.ts [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    initScript()
+
+    const { sdkVersion, apiKey } = settings
+    const version = sdkVersion ?? defaultVersion
+
+    await deps
+      .loadScript(`https://storage.googleapis.com/sdk.getripe.com/sdk/${version}/sdk.umd.js`)
+      .catch((err) => console.error('Unable to load Ripe SDK script', err))
+
+    await analytics.ready(async () => {
```
[the remark this answers is not in the document]
Comment (simpajj) [reply packages/browser-destinations/src/destinations/ripe/index.ts]: No reason, misunderstanding on my part. Your suggestion seems like the more appropriate approach!

File: packages/browser-destinations/src/destinations/ripe/alias/index.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    groupId: {
+      type: 'string',
+      required: false,
+      description: 'The ID associated groupId',
+      label: 'Group ID',
+      default: { '@path': '$.groupId' }
+    }
+  },
+  perform: async (ripe, { payload }) => {
```
[the remark this answers is not in the document]
Comment (simpajj) [reply packages/browser-destinations/src/destinations/ripe/alias/index.ts:34]: Thanks for the heads up! Our SDK currently keeps track of the user id, so we only need the new one. I'll leave the logic as is for now, if that's ok with you guys.

**Pull request merged**